src/shared/components/case-viewer/services/case.resolver.spec.ts

```diff
@@ -222,6 +225,25 @@ describe('CaseResolver', () => {
       expect(navigationNotifierService.announceNavigation).toHaveBeenCalledWith({action: NavigationOrigin.NO_READ_ACCESS_REDIRECTION});
 
     });
+
+    it('should redirect to no results found page when case reference search from header is not found', () => {
+      const error = {
+        status: 400
+      };
+      casesService.getCaseViewV2.and.returnValue(Observable.throw(error));
+
+      router = {
+        navigate: jasmine.createSpy('navigate'),
+        getCurrentNavigation: jasmine.createSpy('getCurrentNavigation'),
+        events: Observable.of( new NavigationEnd(0, '/trigger/COMPLETE/process', '/home'))
```

Comment:
@johnbenjamin-hmcts - extra space between `(` and `new`. Also, I think `Observable.of` can be imported so that it's just `of(...)`, although I'm aware other places in the code are also using `Observable.of(...)`.

Reply:
Thank you @Daniel-Lam, I have now resolved this.